### PR TITLE
Antlr4

### PR DESCRIPTION
**Pull request by muskaan2199**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: a/abyss/abyss_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+../configure --prefix=/usr/local/abyss
+
+if ! make; then
+        echo "Build fails"
+        exit 2
+fi
+
+if ! make install; then
+        echo "Install fails"
+        exit 2
```
Comment (nikhil-kalbande) [inline a/abyss/abyss_ubi_8.5.sh:60]: @MuskaanSheik  we want keep return code "2" for test failure. So for build/install you can keep exit 1 only

File: a/abyss/abyss_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=abyss
+PACKAGE_VERSION=${1:-2.3.5}
+PACKAGE_URL=https://github.com/bcgsc/abyss.git
+
+yum install -y sudo
+sudo yum update -y
```
Comment (nikhil-kalbande) [inline a/abyss/abyss_ubi_8.5.sh:26]: @MuskaanSheik  This script is supported for root or non-root user? why are we using sudo at some places only?

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: a/abyss/abyss_ubi_8.5.sh (hunk 1 above)
Comment (muskaan2199) [reply a/abyss/abyss_ubi_8.5.sh:60]: @nikhil-kalbande made the required change

File: a/abyss/abyss_ubi_8.5.sh (hunk 2 above)
Comment (muskaan2199) [reply a/abyss/abyss_ubi_8.5.sh:26]: @nikhil-kalbande It isn't needed. Removed that. I was following the steps in the abyss repository. But works the same without sudo so removed it

**Pull request closed without merging**